src/operation-node/unique-constraint-node.ts

```diff
@@ -3,10 +3,13 @@ import { ColumnNode } from './column-node.js'
 import { IdentifierNode } from './identifier-node.js'
 import { OperationNode } from './operation-node.js'
 
+export type NullsNotDistinct = 'nulls not distinct' | 'nulls distinct'
+
 export interface UniqueConstraintNode extends OperationNode {
   readonly kind: 'UniqueConstraintNode'
   readonly columns: ReadonlyArray<ColumnNode>
   readonly name?: IdentifierNode
+  readonly nullsNotDistinct?: NullsNotDistinct
 }
```

Comment:
since nulls are distinct by default, only need a flag - `nullsNotDistinct: boolean`

Reply:
Changed and added new options to other queries (please see in the description)